frontend/src/pages/RegistrationsPage.tsx

```diff
@@ -0,0 +1,28 @@
+import {Box, Button, TextField} from "@mui/material";
+import "./RegistrationsPage.css";
+import {FormEvent} from "react";
+
+export default function RegistrationsPage() {
+
+    const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+
+    }
```

Comment:
Diese Methode macht nicht. Wir wollen keine unvollständigen Code im Main branch haben. Also vervollständige das doch oder lass diese Componente weg bis sie fertig ist.

Reply:
ich habe die methode in den anderen Branch fertig geschrieben. werde das bald mergen